### ISSUE TITLE
Add an ability to show views over the keyboard (when it's open)

### ISSUE THREAD

**Issue by kirillzyusko** [kirillzyusko/react-native-keyboard-controller#373]
**Is your feature request related to a problem? Please describe.**

I'd like to have an ability to show custom views over the keyboard as in Telegram, for example:

|iOS|Android|
|---|--------|
|<video src="https://github.com/kirillzyusko/react-native-keyboard-controller/assets/22820318/476165b7-fa56-4a6d-a76c-a43613356343">|<video src="https://github.com/kirillzyusko/react-native-keyboard-controller/assets/22820318/0648e3d2-5ca2-4234-8c6a-77ffbf600fab">|

**Describe the solution you'd like**

That would be good to have `OverKeyboardView` exported from this library. The API could be like:

```tsx
<OverKeyboardView>
  {/* custom views is placed here */}
</OverKeyboardView>
```

**Describe alternatives you've considered**

There is no good alternatives. Right now we need to close a keyboard and show elements to be sure they are not drawn under the keyboard.

Modals and portals are not helping a lot here, since view still obscured by the keyboard.

**Additional context**

We can take telegram app as a reference to see how it was implemented there.

**Comment by kirillzyusko** [comment]
@fukemy @Braden1996 @VladyslavMartynov10 @hirbod @luluapp @ha3 tagging you all as you reacted with 🚀  - I published a new version `1.14.0` where I added `OverKeyboardView` component 👀 

@terrysahaidak tagging you too since at some point of time we discussed it as well 🙃

**Comment by terrysahaidak** [comment]
This is awesome! I'll definitely play with it.
